nix_update/version/github.py

```diff
@@ -21,32 +23,39 @@ def version_from_entry(entry: Element) -> Version:
     # TODO: set pre-release flag
     return Version(unquote(url.path.split("/")[-1]))
 
+def _dorequest(url: ParseResult, feed_url: str) -> str:
+    request = urllib.request.Request(feed_url)
+
+    login, account, password = netrc.netrc().authenticators(url.netloc)
+    if login is not None:
+        info(f"using netrc file")
```

Comment:
```suggestion
        info(f"using netrc file: %s")
```
maybe we should print which one is used if we can get the path here easily?